main.tf

```diff
@@ -8,6 +8,187 @@ terraform {
   }
 }
 
+
 provider "aws" {
   region = "eu-central-1"
 }
```

Comment:
@Davidoutz Usually setting the provider inside the module is not best practice. You just set the required provider but it's up to the "caller" to declare it.

> A module intended to be called by one or more other modules must not contain any provider blocks. A module containing its own provider configurations is not compatible with the for_each, count, and depends_on arguments that were introduced in Terraform v0.13.

source: [Terraform docs](https://developer.hashicorp.com/terraform/language/modules/develop/providers)

Reply:
Well I know…. :wink: but 1. this came from the template so what do we do ? and 2. the the pre-commit action fails afterwards. You can check some of the previous workflows.